docs/xxprintflagsfinal.md

```diff
@@ -0,0 +1,40 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]PrintFlagsFinal
+
+When enabled, this Hotspot option outputs the values of all configuration parameters.
```

Comment:
We should probably also be clear here that only a small subset of options are currently being output by this option. I'm not sure we want to highlight just how minimal the subset is, though (only two options are currently displayed). Otherwise we'll have to keep updating this document if we add new options. Perhaps just mentioning that this option shows only a subset of options found to be necessary, and that new options will be added as we learn of new requirements, or something like that?